woocommerce/wc-api-dev
# Check for REST Controllers Removed in 3.5

Original Bug Report: p1536939575000100-slack-proton

`wc-api-dev` is still needed for AT sites in order for the Store on WPCOM interface to function. And since we don't force an auto update of WooCommerce on AT sites, we need to ensure the plugin is compatible with 3.5 and older versions.

This PR adds in some logic to check for the presence of classes that are being removed in Woo 3.5 to ensure the plugin will operate still under 3.5

__To Test__
- Install this plugin alongside the beta of Woo 3.5
- Verify no errors are thrown
